### PR TITLE
Remove syslog logging

### PR DESCRIPTION
**Pull request by runely**
- Removed `winston-syslog` as **syslog** logging
- Replaced it with `axios` for **token-based** logging

**Token-based** logging uses `basic auth` where **username** is blank and the **password** is the **token** provided from the aggregator.

Replacement of `winston-syslog` needed to be done because it doesn't closes its own TCP ports 🙀  This in time breaks Azure Functions hitting maximum allowed open TCP ports...

And reverting back to `syslog-client` is no-go since it does't work with Azure Functions using `VNet integration`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,10 +28,9 @@
   ],
   "license": "MIT",
   "dependencies": {
+    "axios": "0.24.0",
     "deepmerge": "^4.2.2",
-    "pkg-dir": "5.0.0",
-    "winston": "3.3.3",
-    "winston-syslog": "2.4.4"
```
Comment (jorgtho) [inline package.json:34]: bye bye winston

File: src/lib/format-date-time.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 module.exports = (time) => {
-  const fDate = `${time.getDate()}/${time.getMonth() + 1}/${time.getFullYear()}`
+  const fDate = `${addZero(time.getDate())}.${addZero(time.getMonth() + 1)}.${time.getFullYear()}`
```
Comment (jorgtho) [inline src/lib/format-date-time.js:2]: haha, kosa seg med datoer igjen jo

File: package.json (hunk 1 above)
Comment (runely) [reply package.json:34]: 👋

**Pull request merged**